wallet.go

```diff
@@ -39,6 +40,8 @@ type Wallet struct {
 	cancelFuncs        []context.CancelFunc
 	cancelAccountMixer context.CancelFunc
 
+	cancelAutoTicketBuyerMux sync.Mutex
```

Comment:
Just seeing this properly, `mux` has a different meaning in go code I've seen.
```suggestion
	cancelAutoTicketBuyerMu sync.Mutex
```